test/mailers/notification_mailer_test.rb

```diff
@@ -27,4 +30,17 @@ def test_mail_gets_delivered
     assert_equal(['administrator@auction.test'], email.to)
     assert_equal('Daily summary for 2010-07-04', email.subject)
   end
+
+  def test_sends_auctioned_domain_list_to_subscribers
+    perform_enqueued_jobs do
+      DailyBroadcastAuctionsJob.perform_now
```

Comment:
Here, a job is tested in mailer test. Might be something like
```
email = ContactMailer.email_changed(contact: @contact, old_email: 'john-old@inbox.test')
              .deliver_now
```

More examples: https://github.com/internetee/registry/tree/master/test/mailers